### PR TITLE
README: Fix outdated package-configuration helper-cli documentation

### PR DESCRIPTION
**Pull request by MarcelBochtler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ or rule violations resolutions in the context of the repository.
 
 A single file or a directory with multiple files containing configurations to set provenance-specific path excludes and
 license finding curations for dependency packages to address issues found within a scan result. The `helper-cli`'s
-[GeneratePackageConfigurationsCommand](./helper-cli/src/main/kotlin/commands/GeneratePackageConfigurationsCommand.kt)
+[package-configuration create command](./helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt)
```
Comment (sschuberth) [inline README.md:289]: Nit: Maybe write `*create*` to make the command name stand out in italics?

Comment (MarcelBochtler) [reply README.md:289]: The whole sub-command is `package-configuration create`. I'll use italics for this.

Comment (sschuberth) [reply README.md]: Looks like exactly the same things was done in https://github.com/oss-review-toolkit/ort/pull/4190 now by @fviernau.

Comment (fviernau) [reply README.md]: Oh sorry, I hadn't seen this one early enough.

**Pull request closed without merging**